charts/topolvm/README.md

```diff
@@ -104,6 +104,14 @@ You need to configure kube-scheduler to use topolvm-scheduler extender by referr
 | controller.tolerations | list | `[]` | Specify tolerations. # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/ |
 | controller.updateStrategy | object | `{}` | Specify updateStrategy. |
 | controller.volumes | list | `[{"emptyDir":{},"name":"socket-dir"}]` | Specify volumes. |
+| env.csi_provisioner | list | `[]` | Specify environment variables for csi_provisioner container. |
+| env.csi_registrar | list | `[]` | Specify environment variables for csi_registrar container. |
+| env.csi_resizer | list | `[]` | Specify environment variables for csi_resizer container. |
+| env.csi_snapshotter | list | `[]` | Specify environment variables for csi_snapshotter container. |
+| env.liveness_probe | list | `[]` | Specify environment variables for liveness_probe container. |
+| env.topolvm_controller | list | `[]` | Specify environment variables for topolvm_controller container. |
+| env.topolvm_node | list | `[]` | Specify environment variables for topolvm_node container. |
+| env.topolvm_scheduler | list | `[]` | Specify environment variables for topolvm_scheduler container. |
```

Comment:
The other parameters are camel cases. It is better to be the same.

Reply:
Certainly most fields are camelCase, but it would be more consistent to make an exception if it represents a container identifier.
 
https://github.com/topolvm/topolvm/blob/d242dcc522ae3f4bc3fe692054205b1076319396/charts/topolvm/README.md?plain=1#L161-L169
https://github.com/topolvm/topolvm/blob/d242dcc522ae3f4bc3fe692054205b1076319396/charts/topolvm/README.md?plain=1#L116-L120